jbpm-case-mgmt/jbpm-case-mgmt-impl/src/test/resources/cases/UserTaskCaseRestrictedCaseFileItem.bpmn2

```diff
@@ -18,6 +18,14 @@
         </tns:metaData>
       </bpmn2:extensionElements>
     </bpmn2:property>
+    
+    <bpmn2:property id="caseFile_r" itemSubjectRef="_sItem" name="caseFile_r">
```

Comment:
generate new file for these test.

Reply:
Duplicating bpmn definitions (as duplicating code through copy paste), does not seem a good idea. The modified file was initially created to check required constraint. It makes full sense to reuse it to check readonly constraint.